transcom/mymove
# MB-16933 Update NS Mayport, FL 32233 to mark it as having government counseling

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16933)

## Summary

Changes the `provides_services_counseling` value for the duty location to `true`.

Is there anything you would like reviewers to give additional scrutiny?

## Verification Steps for the Author

These are to be checked by the author.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Run migrations using make db_dev_migrate.
2. Look at the duty location with this id: `3bfe8068-f9d1-4c21-ac19-1c0a60dcd326`.
3. See that it's Mayport and that `provides_services_counseling` is set to `true`.
